schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -73,6 +74,19 @@ conditional_schedule:
         - yast2_gui/yast2_datetime
         - yast2_gui/yast2_hostnames
 test_data:
+  mistakes:
```

Comment:
I wonder if we can come up with a better naming. Maybe `misconfiguration`, or `typos` which sound better to me. We don't have to change it.

Reply:
for me it helps more 'misconfiguration', specially in the name of the test

Reply:
I changed the name to be more specific.